plugins/auth-node/src/proxy/types.ts

```diff
@@ -19,18 +19,18 @@ import { Request } from 'express';
 import { ProfileTransform } from '../types';
 
 /** @public */
-export interface ProxyAuthenticator<TContext, TResult> {
+export interface ProxyAuthenticator<TContext, TResult, TProviderInfo> {
```

Comment:
Let's provide a default for `TProviderInfo` to avoid breaking existing users of the type. I think we can default it to `undefined`.